### PR TITLE
Add support for multiple distributions

### PR DESCRIPTION
**Pull request by zoresvit**
Each image has it's own Dockerfile (Debian is the default).  To build testing image based on CentOS remember to specify `Dockerfile.centos` as follows:
```
$ docker build -f Dockerfile.centos -t pythonista-yum .
```
Script `get-python.sh` is now extended to be distribution-agnostic via
shell `case` statements. You can add support for more distributions by
complementing the cases, but don't forget to test the result. For now
Debian and CentOS are supported.

### CODE REVIEW

File: Dockerfile [hunk 1]
```diff
@@ -30,11 +30,11 @@ RUN ["/bin/bash", "/var/tmp/get-python.sh", \
 # to setup it directly in Dockerfile, but it's awful when the
 # PATH should be changed dynamically based on some third-party
 # input. But I have no choice... :(
-ENV PATH $PATH:/opt/python/2.7.10/bin
-ENV PATH $PATH:/opt/python/3.4.3/bin
 ENV PATH $PATH:/opt/python/2.6.9/bin
+ENV PATH $PATH:/opt/python/2.7.10/bin
```
Comment (ikalnitsky) [inline Dockerfile]: Why did you change path order? IIRC, I did such order on purpose - I wanted `python` to be  `python2.7`, and `python3`  to be `python3.4`.

File: Dockerfile.centos [hunk 2]
```diff
@@ -0,0 +1,42 @@
+#######################################################################
+# Dockerfile for building Pythonista docker image.
+#
+# Copyright (c) 2014, Igor Kalnitsky <igor@kalnitsky.org>
```
Comment (ikalnitsky) [inline Dockerfile.centos]: I think you can change the copyright notice too :)

File: README.md [hunk 3]
```diff
@@ -13,8 +13,8 @@ handle it, since Debian Jessie doesn't have Python 2.6 in its repo.
 About Image
 -----------
 
-* **OS** - Debian Jessie (Testing)
-* **CPython** - 2.6.9 / 2.7.10 / 3.2.6 / 3.3.6 / 3.4.3
+* **OS** - Debian Jessie (Testing) or CentOS 7
```
Comment (ikalnitsky) [inline README.md]: Jessie isn't Testing anymore. Please fix also.

File: get-python.sh [hunk 4]
```diff
@@ -60,8 +97,18 @@ function main {
...
 function install_packages {
-    apt-get update
-    apt-get -y install "${@}"
+case ${OSNAME} in
+    "debian")
+        apt-get update
+        apt-get -y install "${@}"
+        ;;
+    "centos")
+        yum -y install epel-release
```
Comment (ikalnitsky) [inline get-python.sh]: Could you please add some comments here? I thought for CentOS having

    yum -y install

is more than enough.

File: Dockerfile (hunk 1 above)
Comment (zoresvit) [inline Dockerfile]: My bad, got confused on that one. Will fix.

File: get-python.sh (hunk 4 above)
Comment (zoresvit) [inline get-python.sh]: No, default repositories have almost nothing, not even `python-pip`. That's why `epel-release` is needed. If you are about `groupinstall development` then yes, I could track required packages instead of installing all development tools, though it may be some hassle.
